<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>351</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Georgia</family>
    <pointsize>12</pointsize>
    <italic>false</italic>
    <stylestrategy>PreferAntialias</stylestrategy>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>HashCheck</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Assets.qrc">
    <normaloff>:/icons/appIcon.png</normaloff>:/icons/appIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">Widget{
	color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="selectionBox">
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Select File and Hash Funtion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="hashFuncLabel">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Hash Function :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="hashCombo">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sFileLabel">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select File :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="sFileButton">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="calculateHash">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Calculate Hash</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="infoBox">
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>File Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLabel" name="filename">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="fNameLabel">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>File Name :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fsizeLabel">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>File Size :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="fileSize">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="hashBox">
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Hash</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="fHashLabel">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>File Hash :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="vHash">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Verify Hash :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="checkButton">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="inputHash">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="fileHash">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="hashMatchLabel">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Hash Match :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="hashMatchStatus">
        <property name="font">
         <font>
          <family>Georgia</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Assets.qrc"/>
 </resources>
 <connections/>
</ui>
